### DISCUSSION TITLE
Demo Page/Project Request

### DISCUSSION THREAD

**Discussion by caglarkochan** [livewire/flux#944] [Feature Requests]
Greetings,

There are no demo projects on the page with https://fluxui.dev/demos. 

Could I be the only one experiencing this problem? 

Thank you very much for your support.

Also, is it possible to see a demo with a full implementation like in livewire screencasts?

**Comment by joshhanley** [comment]
@caglarkochan thanks for reporting! This seems like an issue to me not a feature request. Please open issues in the future. I've submitted a PR to the docs to fix this.

**Comment by Kh4os-afk** [comment]
haha I didn't even know this page existed

**Comment by kyranb** [comment]
@joshhanley Are code examples for these demos available anywhere?

**Reply by joshhanley** [reply]
@kyranb not at this stage. Officially the demos page doesn't exist yet 😆 hence why it's not linked too from the docs
